themes/developer/developer.theme.sh

```diff
@@ -54,12 +54,24 @@ function _omb_theme_developer__read_ruby_version {
 ## @var[out] REPLY
 function _omb_theme_developer__readPyVersion {
   local val_py=$(python --version 2>/dev/null | cut -d ' ' -f 2)
+  local val_uv=$(uv --version 2>/dev/null | cut -d ' ' -f 2)
+  local val_venv=""
+
+  ## according some docs this env var just exist when a venv/uv venv is active
+  if [[ -n "$VIRTUAL_ENV" ]]; then
+    val_venv=" ($(echo "$VIRTUAL_ENV" | awk -F/ '{print $(NF-1)"/"$NF}'))"
```

Comment:
I'm wondering what happens when `VIRTUAL_ENV` contains a newline in the path. The above would output multiple `aaa/bbb` pairs. Also, what happens when `VIRTUAL_ENV` doesn't contain any `/`? It would output `/` + `$VIRTUAL_ENV`, but I'm not sure if it's the intended behavior. I'd suggest simply doing

```suggestion
  if [[ $VIRTUAL_ENV =~ [^/]+/[^/]+$ ]]; then
    val_venv=" ($BASH_REMATCH)"
```

Reply:
> Also, what happens when `VIRTUAL_ENV` doesn't contain any `/`?

This hasn't been answered. Or is it ensured that it `VIRTUAL_ENV` contains `/`?